ta/Makefile

```diff
@@ -5,6 +5,8 @@ ifeq ($O,)
 $(error output path should be specified when calling this makefile)
 endif
 
+LDFLAGS=
```

Comment:
As I have replied before also make LDFLAGS= or LDFLAGS= make fails to compile client application as it fails to load standard headers needed to compile it.